swan-lake/integration-tools/openapi-tool.md

```diff
@@ -942,3 +942,201 @@ The attributes of the annotation are optional and can be used for each particula
 | `licenseName: string`       | You can use this to add the name of the license under which the API is provided.                                                                                                                                                                                                                                                          | Optional          |
 | `licenseURL: string`        | You can use this to add the URL details regarding the full text of the license.                                                                                                                                                                                                                                                           | Optional          |
 | `embed: string`             | Turns on generating OpenAPI documentation for the service for [introspection endpoint](https://ballerina.io/spec/http/#236-openapi-specification-resources) support when used with `true` in the annotation.                                                                                                                                                                                               | Optional          |
+
```

Comment:
We need to update the following sections as well right?
- [Usage](https://github.com/ballerina-platform/ballerina-dev-website/blob/master/swan-lake/integration-tools/openapi-tool.md#usage)
- [Command Options](https://github.com/ballerina-platform/ballerina-dev-website/blob/master/swan-lake/integration-tools/openapi-tool.md#command-options)

Reply:
with regarding to the `sanitize`, `flatten` subcommand or `--with-service-type` flag ?

Reply:
Regarding the sub commands

Reply:
Fixed via [0ffae86](https://github.com/ballerina-platform/ballerina-dev-website/pull/9626/commits/0ffae869cfe37038505105475a276d0c539f066c)